README.md

```diff
@@ -292,10 +302,22 @@ deactivate
 
 ## Release notes
 
+### Version 0.5.2
+
+New features:
+
+- DER fix
```

Comment:
-  Allow EC signatures in DER format from signers and verify signature format during validation.
- Fixes bug in signing BMFF formats.
- Adds the ability to verify PDF files (but not to sign them).
- 2x or better speed improvement on sign_file when using file paths (uses native Rust file I/O).
- fixes for RIFF and GIF formats.